lms/static/scripts/frontend_apps/utils/api.js

```diff
@@ -4,7 +4,8 @@
  */
 export class ApiError extends Error {
   constructor(status, data) {
-    const message = data.error_message || data.message || 'API call failed';
```

Comment:
The default value of "API call failed" was provided so that if code which doesn't know about API errors specifically ends up receiving this error it will have a useful `message` property. That is distinct from `errorMessage` which exists to allow callers who are specifically aware of `ApiError` to extract just the message that came from the server.